File: C Modules/Assignments/Assignment no 6/Question-1/Type_2/Assignment no 1/Question_2.c
#include<stdio.h>
int palindrome()
{
    int num,first,last;
    printf("Enter the 3-digit number:");
    scanf("%d",&num);
    first=num/100;
    last=num%10;
    if(num>=100 && num<=999 && first==last){
        return 1;
    }
    else{
        return 0;
} 
}
int main()
{
    int num,first,last;
    if(palindrome()){
    printf("it is palindrome number");
    }
    else{
    printf("it is not palindrome number");
}
    return 0;
}

File: Java Module/Assignments in c to java/Assignment-01/Question_2.java
public class Question_2 {

    public static void main(String[] args) {
            int num=121,first,last;
    first=num/100;
    last=num%10;
    if(num>=100 && num<=999 && first==last){
    System.out.printf("%d it is palindrome number",num);
    }
    else{
    System.out.printf("%d it is not palindrome number",num);
}
    }
}